tests/schema/tests.py

```diff
@@ -1211,6 +1233,22 @@ def test_alter_int_pk_to_bigautofield_pk(self):
         with connection.schema_editor() as editor:
             editor.alter_field(IntegerPK, old_field, new_field, strict=True)
 
+    def test_alter_int_pk_to_smallautofield_pk(self):
+        """
+        Should be able to rename an IntegerField(primary_key=True) to
+        SmallAutoField(primary_key=True).
+        """
```

Comment:
I changed this to `SmallIntegerField` -> `SmallAutoField` conversion. Generally, changing `Integer` to `SmallAuto` is not sth that we want to support.

Reply:
Good catch. (While possible if the stored values are in range it is highly unusual to do that.)